new-frontiers-14/frontier-station-14
# Adds the CiviMed Vending Machine

High priced medical supplies for long duration playerbase needs.




## About the PR



Adds:
New vending machine with a high price modifier. Visuals are a green redress of the conventional medical vendor.
Restock crates with unique graphics.
Unique suite of objectively hilarious advertisements.

## Why / Balance


This vending machine is designed to sell topical medications (gauze, ointment, bruise packs and blood packs) at high above normal market values in order to provide a relief valve for convenience versus time for low pop rounds and to provide medical merchants with a price point to operate within.

## How to test


Spawn vending machine.
Check inventory. Make a purchase.
Open machine with screwdriver.
Spawn restock crate. 
Restock vending machine.
Close vending machine.
Check inventory. Make a purchase.
Break the vending machine. I like using an axe.

## Media


![civimed](https://github.com/new-frontiers-14/frontier-station-14/assets/54123313/74347e47-007b-4cfa-a59a-5713a77a36ef)

- [ X ] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes


None.

**Changelog**
